accounts/usbwallet/hub.go

```diff
@@ -93,30 +94,30 @@ func NewLedgerHub() (*Hub, error) {
 		0x4011, /* HID + WebUSB Ledger Nano X */
 		0x5011, /* HID + WebUSB Ledger Nano S Plus */
 		0x6011, /* HID + WebUSB Ledger Nano FTS */
-	}, 0xffa0, 0, newLedgerDriver)
+	}, []uint16{0xffa0, 0}, 2, newLedgerDriver)
```

Comment:
Question: here the `endpointID` was changed to `2`. Why?

Reply:
Those values matches what was enumerated on my device.

Reply:
On macos13 I don't see any difference after removing the `usageID` of zero and setting `endpointID` to 0.